docs/en/administration/sync_accounts_between_multiple_hosts.md

```diff
@@ -103,10 +103,10 @@ PLAY RECAP *********************************************************************
 172.16.255.180             : ok=1    changed=0    unreachable=0    failed=1
 ```
 
-In above example,  the group ID 1000 has been allocated to another group on host `172.16.255.180` , we should **change the GID**  or **delete the group with GID 1000** on host `172.16.255.180` , then run the playbook again.
+In the above example, the group ID 1000 has been allocated to another group on host `172.16.255.180`. So we should **change the GID**  or **delete the group with GID 1000** on host `172.16.255.180`, and then run the playbook again.
 
 :::caution
-If the user account has already existed on the host and we change it to another UID or GID value, the user may loss permissions to the files and directories which they previously have. For example:
+If a user account has already existed on the host and we change it to another UID or GID value, the user may lose permissions to the files and directories which they previously had. For example:
```

Comment:
If UID / GID has been changed on an existing user, the user may lose permissions to previously accessible files.

Reply:
```suggestion
If the UID / GID of an existing user is changed, the user may lose permissions to previously accessible files. For example:
```